auto-dark.el

```diff
@@ -65,6 +65,12 @@ end tell")))
 
   (string-equal "true" (string-trim (shell-command-to-string "osascript -e 'tell application \"System Events\" to tell appearance preferences to return dark mode'"))))
 
+(defvar auto-dark-dark-mode-hook nil
+  "List of hooks to run after dark mode is loaded." )
+
+(defvar auto-dark-light-mode-hook nil
+  "List of hooks to run after dark mode is loaded." )
```

Comment:
I think you meant *light* instead of dark here. Right?

Reply:
Right, will fix